vpr/src/place/place_log_util.h

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
Same comment as the definition file. A doxygen comment for the file would be useful!